xml/System.Security.Cryptography/SHA256.xml

```diff
@@ -160,6 +158,17 @@
       </AssemblyInfo>
       <Docs>
         <summary>Allows specific implementations of this abstract class to be instantiated.</summary>
+        <remarks>
+          <format type="text/markdown"><![CDATA[  
+  
+## Remarks  
+ Out of the box, the valid implementations names, and their associated implementations, are the following : 
+ * SHA256, SHA-256 or System.Security.Cryptography.SHA256 : implementation is <see cref="T:System.Security.Cryptography.SHA256Managed" />   
```

Comment:
This is good for a table:
 You can use a table:
| Implementation | hashName |
|----------|---------|
|\<xref:...> | `SHA256` \</br> `SHA-256` \</br> etc... |